html/elements/template.json

```diff
@@ -70,7 +77,7 @@
               "webview_android": "mirror"
             },
             "status": {
-              "experimental": false,
+              "experimental": true,
```

Comment:
Should experimental stay `false`? We've shipped the new behavior, and it's on track for standardization and shipment in other browsers.

Reply:
BCD sets experimental status based upon how many engines a feature is _currently_ implemented in (without flags or origin trials and in a stable release) -- one stable engine is experimental, two stable engines or more is not.  See https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines/index.md#setting-experimental for more details.